core/contracts/oracle/interfaces/RegistryInterface.sol

```diff
@@ -7,40 +7,52 @@ pragma experimental ABIEncoderV2;
  */
 interface RegistryInterface {
     /**
-     * @dev Registers a new derivative. Only authorized derivative creators can call this method.
+     * @notice Registers a new derivative.
```

Comment:
I made a bunch of comments on the comments in `Registry` last week, but the changes in those comments aren't reflected here. Could you look at the changes in those comments and apply them to this interface too, since all the methods/comments are basically the same.

Reply:
sure. on it

Reply:
I will go through all interfaces and be sure that the commenting is consistent.